cl/people_db/import_judges/populate_fjc_judges.py

```diff
@@ -641,3 +641,73 @@ def make_federal_judge(item, testing=False):
                 position.save()
             except Exception:
                 continue
+
+
+def make_mag_bk_judge(item, testing=False):
+    """Takes the federal judge data <item> and associates it with a Judge object.
```

Comment:
It'd be very helpful to have a couple lines of sample data here, so we know what we're processing below.